card_live_dashboard/layouts/__init__.py

```diff
@@ -283,6 +283,11 @@ def figures_layout(figures_dict: Dict[str, go.Figure]):
                                      value_color='default'
                                  ),
                                  ),
+            single_figure_layout(title='RGI intersections',
```

Comment:
I like the idea of creating a separate drop-down for the RGI Intersections. This should mostly consist of adding the following lines of code to this function call (e.g., calling `figures_menu_layout`):

https://github.com/arpcard/card-live-dashboard/blob/fd96fe45a8770b5e0bb66be4b07d3287e3a4503d/card_live_dashboard/layouts/__init__.py#L268-L276

I don't think it makes as much sense (or seems quite complicated) to let people color the bars by an organism/geographic region, so the color part of this function call can be ignored (so you don't have any drop-down menu to select colors).

Once that's done, you would then have to add another `Input` type to the callbacks for the selected RGI intersections type:

https://github.com/arpcard/card-live-dashboard/blob/fd96fe45a8770b5e0bb66be4b07d3287e3a4503d/card_live_dashboard/callbacks/__init__.py#L102

And propagate this value all the way to where you call the `figures.rgi_intersection_figure` function:

https://github.com/arpcard/card-live-dashboard/blob/fd96fe45a8770b5e0bb66be4b07d3287e3a4503d/card_live_dashboard/callbacks/__init__.py#L301

I think that would be all the changes you'd need.

Reply:
Done